plugins/nvm/nvm.plugin.sh

```diff
@@ -1,12 +1,68 @@
 #! bash oh-my-bash.module
+# Description: automatically load nvm 
+#
+# @var[opt] OMB_PLUGIN_NVM_AUTO_USE enable .nvmrc autoload
 
 # Set NVM_DIR if it isn't already defined
 [[ -z "$NVM_DIR" ]] && export NVM_DIR="$HOME/.nvm"
 
 
 # Try to load nvm only if command not already available
 if ! _omb_util_command_exists nvm; then
-  [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
-  # This is done as part of completions!!!
-  # [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+  # shellcheck source=/dev/null
+  [[ -s $NVM_DIR/nvm.sh ]] && source "$NVM_DIR/nvm.sh"
+fi
+
+# Optional .nvmrc autoload
+if _omb_util_command_exists nvm && [[ ${OMB_PLUGIN_NVM_AUTO_USE-} == true ]]; then
+  _omb_plugin_nvm_find_up () {
+    local path=$PWD
+    while [[ $path && ! -e $path/$1 ]]; do
+        path=${path%/*}
+    done
+    echo "$path"
+  }
+  
+  cdnvm(){
+    cd "$@" || return "$?"
+    local nvm_path=$(_omb_plugin_nvm_find_up .nvmrc)
+
+    # If there are no .nvmrc file, use the default nvm version
+    if [[ $nvm_path != *[^[:space:]]* ]]; then
+
+      local default_version
+      default_version=$(nvm version default)
+
+      # If there is no default version, set it to `node`
+      # This will use the latest version on your machine
+      if [[ $default_version == "N/A" ]]; then
+          nvm alias default node
+          default_version=$(nvm version default)
+      fi
+
+      # If the current version is not the default version, set it to use the default version
+      if [[ $(nvm current) != "$default_version" ]]; then
+          nvm use default
+      fi
+
+    elif [[ -s $nvm_path/.nvmrc && -r $nvm_path/.nvmrc ]]; then
+      local nvm_version
+      nvm_version=$(<"$nvm_path"/.nvmrc)
```

Comment:
```suggestion
      nvm_version=$(<"$nvm_path"/.nvmrc)
```

This is really cosmetic but I would like to add a space here. This is just my preference, but what do you think?